docs/sources/setup-grafana/configure-security/configure-authentication/saml/index.md

```diff
@@ -135,7 +140,13 @@ For Grafana Cloud instances, please contact Grafana Support to update the `conte
 
 The SAML SSO standard uses asymmetric encryption to exchange information between the SP (Grafana) and the IdP. To perform such encryption, you need a public part and a private part. In this case, the X.509 certificate provides the public part, while the private key provides the private part. The private key needs to be issued in a [PKCS#8](https://en.wikipedia.org/wiki/PKCS_8) format.
 
-Grafana supports two ways of specifying both the `certificate` and `private_key`.
+{{% admonition type="note" %}}
+Directly supplying a certificate and private key to Grafana is optional. Commonly, the certificate and key are embedded in the [IDP metadata](#configure-the-saml-toolkit-application-endpoints) and refreshed as needed by Grafana automatically. This can save you the work of manually rotating your certs and keys whenever they expire.
```

Comment:
> Directly supplying a certificate and private key to Grafana is optional. Commonly, the certificate and key are embedded in the [IDP metadata](#configure-the-saml-toolkit-application-endpoints)

The cert and private key that could be configured on the Grafana side are different from the certificate (and the public key) in the IDP metadata, so I think it should be rewritten to convey the message that it should be configured if their IdP expects signed requests.